json_schemas/services-g-cloud-9-cloud-software.json

```diff
@@ -2,18 +2,348 @@
+        "No data sanitisation process"
+      ]
+    },
+    "dataStorageAndProcessingLocations": {
+      "items": {
+        "enum": [
+          "United Kingdom",
+          "European Economic Area (EEA)",
+          "EU-US Privacy Shield agreement locations",
+          "Other locations"
+        ]
+      },
+      "maxItems": 4,
+      "minItems": 1,
+      "type": "array",
+      "uniqueItems": true
+    },
+    "datacentreSecurityStandards": {
+      "enum": [
+        "Complies with a recognised standard, for example CSA CCM v3.0 or SSAE-16 / ISAE 3402",
+        "Supplier-defined controls",
+        "Undisclosed"
+      ]
+    },
+    "devicesForManagement": {
+      "items": {
+        "enum": [
+          "Dedicated device on a segregated network (providers own provision)",
+          "Dedicated device on a government network (for example PSN)",
+          "Dedicated device over multiple services or networks",
+          "Any device but through a bastion host",
+          "Directly from any device which may also be used for normal business"
+        ]
+      },
+      "maxItems": 5,
+      "minItems": 1,
+      "type": "array",
+      "uniqueItems": true
+    },
+    "documentationFormats": {
+      "items": {
+        "enum": [
+          "HTML",
+          "ODF",
+          "PDF",
+          "Other"
+        ]
+      },
+      "maxItems": 4,
+      "minItems": 1,
+      "type": "array",
+      "uniqueItems": true
+    },
+    "educationPricing": {
+      "type": "boolean"
+    },
+    "emailSupport": {
+      "type": "boolean"
+    },
+    "endOfContractDataExtraction": {
+      "minLength": 1,
+      "type": "string"
+    },
+    "equipmentDisposalApproach": {
+      "enum": [
+        "Complying with a recognised standard, for example CSA CCM v.30 or ISO/IEC 27001",
+        "Complying with NCSC or CESG Assured Service (Destruction) scheme",
+        "In-house destruction process",
+        "A third-party destruction service",
+        "Undisclosed"
+      ]
+    },
+    "gettingStarted": {
+      "minLength": 1,
+      "type": "string"
+    },
+    "governmentWANS": {
+      "items": {
+        "enum": [
+          "Public Services Network (PSN)",
+          "Police National Network (PNN)",
+          "New NHS Network (N3)",
+          "Joint Academic Network (JANET)",
+          "Scottish Wide Area Network (SWAN)"
+        ]
+      },
+      "maxItems": 5,
+      "minItems": 1,
+      "type": "array",
+      "uniqueItems": true
+    },
+    "guaranteedAvailability": {
+      "minLength": 1,
+      "type": "string"
+    },
+    "howSharedInfrastructureIsKeptSeparate": {
+      "minLength": 1,
+      "type": "string"
+    },
+    "incidentManagementApproach": {
+      "minLength": 1,
+      "type": "string"
+    },
+    "incidentManagementType": {
+      "enum": [
+        "Conforms to a recognised standard, for example, CSA CCM v3.0 or ISO/IEC 27035:2011 or SSAE-16 / ISAE 3402",
```

Comment:
Should we have a think about whether it would be better to store short values rather than the entire label text for some of these checkboxes and radio button questions?  I know it's what we have at the moment, but it feels like we're missing a trick.